LogicalTypes.md

```diff
@@ -741,18 +746,19 @@ keys.
 It is required that the repeated group of key-value pairs is named `key_value`
 and that its fields are named `key` and `value`. However, these names may not
 be used in existing data and should not be enforced as errors when reading.
+(`key` and `value` can be identified by their position in case of misnaming.)
 
 Some existing data incorrectly used `MAP_KEY_VALUE` in place of `MAP`. For
 backward-compatibility, a group annotated with `MAP_KEY_VALUE` that is not
 contained by a `MAP`-annotated group should be handled as a `MAP`-annotated
-group.
+group. `MAP_KEY_VALUE` may be used for the `key_value` group.
 
 Examples that can be interpreted using these rules:
 
 ```
 // Map<String, Integer> (nullable map, non-null values)
 optional group my_map (MAP) {
```

Comment:
Is it possible to have the below one:
```
optional group my_map {
  repeated group map (MAP_KEY_VALUE) {
    required binary str (STRING);
    required int32 num;
  }
}
```

Reply:
No, I don't think so. At the topmost element we need either a `MAP` or a `MAP_KEY_VALUE` (for backward compatibility).

Reply:
This new sentence should not change anything. Just wanted to highlight why it was originally invented. If it causes confusion, it does not worth it.
(At the repeated level we do not say anything about its potential logical type. So, a reader should ignore anything there anyway.)

Reply:
@wgtmac, do you think this sentence is misleading? I'm happy to remove it. The implementation do not need to check the logical type of the repeated field anyway.

Reply:
I think it is. Especially when we have said that the name of `key_value` is not required in the previous paragraph.